gtk/src/rt.rs

```diff
@@ -117,6 +117,12 @@ pub fn init() -> Result<(), glib::BoolError> {
         let argv = ::std::env::args().take(1).collect::<Vec<_>>();
 
         if from_glib(ffi::gtk_init_check(&mut 1, &mut argv.to_glib_none().0)) {
+            let result: bool = from_glib(glib::ffi::g_main_context_acquire(
```

Comment:
Can you add a comment here *why* this is done?

Reply:
Would a short description like this suffice?
```
# See https://github.com/gtk-rs/gtk-rs-core/issues/186 for reasoning behind acquiring and leaking the main context here.
```
Or do you want a longer one?

Reply:
Done in https://github.com/gtk-rs/gtk3-rs/commit/7f36de09e73e4103a916b3d62f7f0af56131ab70